src/app/(social-page)/events/page.tsx

```diff
@@ -197,12 +197,14 @@ export default function Page() {
           </div>
         </div>
       </div>
-      <Pagination
-        totalPages={totalPages}
-        curPage={curPage}
-        onNextPage={onNextPage}
-        onPrevPage={onPrevPage}
-      />
+      {curPage > 1 && (
+        <Pagination
+          totalPages={totalPages}
+          curPage={curPage}
+          onNextPage={onNextPage}
+          onPrevPage={onPrevPage}
+        />
+      )}
```

Comment:
```suggestion
      {totalPages > 1 && (
        <Pagination
          totalPages={totalPages}
          curPage={curPage}
          onNextPage={onNextPage}
          onPrevPage={onPrevPage}
        />
      )}
```
muốn ko hiện phân trang khi chỉ có 1 trang thì cái cần đưa vô điều kiện là totalPages
chỉnh lại toàn bộ ở đưới lun